src/map/status.cpp

```diff
@@ -3151,6 +3172,8 @@ static int status_get_hpbonus(struct block_list *bl, enum e_status_bonus type) {
 				bonus -= (4 * sc->data[SC_GT_CHANGE]->val1);
 			if(sc->data[SC_EQC])
 				bonus -= sc->data[SC_EQC]->val3;
+			if (sc->data[SC_LUNARSTANCE])
+				bonus += sc->data[SC_LUNARSTANCE]->val2;
```

Comment:
this bit is in the (human) logical ordering. the later block is for decreasing HP bonuses, increasing would move it up about 15 lines!